js/content/common.js

```diff
@@ -1209,24 +1209,32 @@ let AugmentedSteam = (function() {
         // Remove Steam's back-to-top button
         DOMHelper.remove("#BackToTop");
 
-        HTML.afterBegin("body", `<div class="es_btt">&#9650;</div>`);
+        let btn = document.createElement("div");
+        btn.classList.add("es_btt");
+        btn.textContent = "▲";
+        btn.style.visibility = "hidden";
 
-        let node = document.querySelector(".es_btt");
+        document.body.append(btn);
 
-        node.addEventListener("click", () => {
+        btn.addEventListener("click", () => {
             window.scroll({
                 top: 0,
                 left: 0,
                 behavior: 'smooth'
             });
         });
 
-        window.addEventListener("scroll", opacityHandler);
+        btn.addEventListener("transitionend", () => {
+            if (btn.style.visibility === "hidden") {
+                btn.style.visibility = "visible";
+            } else {
+                btn.style.visibility = "hidden";
+            }
+        });
```

Comment:
This might work better
````
        btn.addEventListener("transitionstart", () => {
            if (btn.style.visibility === "hidden") {
                btn.style.visibility = "visible";
            } else {
                setTimeout(() => {
                    btn.style.visibility = "hidden";
                }, 200);
            }
        });
````
but it seems like `transitionstart` isn't widely supported yet: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/transitionstart_event

Reply:
Our minimum supported version of FF is 53.0, so we should be good to go unless there are other restrictions